packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -59,6 +59,45 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
       })
     })
 
+    it('can translate, create, and publish a draft of a policy intent', async () => {
+      const draft: NoParkingIntentDraft = {
+        intent_type: 'no_parking',
+        rule_fields: {
+          geographies: [],
+          days: null,
+          start_time: null,
```

Comment:
Might want to just put a placeholder date value here and assert it just to catch any potential bugs associating start_time in the intent to the policy